geode-core/src/test/java/org/apache/geode/management/internal/cli/domain/RegionDescriptionJUnitTest.java

```diff
@@ -0,0 +1,312 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.json.JSONObject;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import org.apache.geode.management.cli.Result.Status;
+import org.apache.geode.management.internal.cli.commands.DescribeRegionCommand;
+import org.apache.geode.management.internal.cli.result.CommandResult;
+import org.apache.geode.test.junit.categories.UnitTest;
+
+@Category(UnitTest.class)
+public class RegionDescriptionJUnitTest {
+  private static final String evictionKeyShared = "sharedEvictionKey";
+  private static final String partKeyShared = "sharedPartitionKey";
+  private static final String regKeyShared = "sharedRegionKey";
+  private static final String evictionValueShared = "sharedEvictionValue";
+  private static final String partValueShared = "sharedPartitionValue";
+  private static final String regValueShared = "sharedRegionValue";
+
+  private static final String evictionKeyA = "uniqueEvictionKey_A";
+  private static final String partKeyA = "uniquePartitionKey_A";
+  private static final String regKeyA = "uniqueRegionKey_A";
+  private static final String evictionValueA = "uniqueEvictionValue_A";
+  private static final String partValueA = "uniquePartitionValue_A";
+  private static final String regValueA = "uniqueRegionValue_A";
+
+  private static final String evictionKeyB = "uniqueEvictionKey_B";
+  private static final String partKeyB = "uniquePartitionKey_B";
+  private static final String regKeyB = "uniqueRegionKey_B";
+  private static final String evictionValueB = "uniqueEvictionValue_B";
+  private static final String partValueB = "uniquePartitionValue_B";
+  private static final String regValueB = "uniqueRegionValue_B";
+
+  public static final String regionName = "mockRegion1";
+
+  private static final DescribeRegionCommand command = spy(DescribeRegionCommand.class);
+
+  @Test
+  public void findCommonRemovesUnsharedKeys() {
+    Map<String, String> commonMap = new HashMap<>();
+    commonMap.put(evictionKeyShared, evictionValueShared);
+    commonMap.put(partKeyShared, partValueShared);
+    commonMap.put(regKeyShared, regValueShared);
+    commonMap.put(evictionKeyA, evictionValueA);
+    commonMap.put(partKeyA, partValueA);
+
+    Map<String, String> comparisonMap = new HashMap<>();
+    comparisonMap.put(evictionKeyShared, evictionValueShared);
+    comparisonMap.put(partKeyShared, partValueShared);
+    comparisonMap.put(regKeyShared, regValueShared);
+    commonMap.put(evictionKeyB, evictionValueB);
+    commonMap.put(regKeyA, regValueB);
+
+    findCommon(commonMap, comparisonMap);
+
+    assertThat(commonMap).containsOnlyKeys(evictionKeyShared, partKeyShared, regKeyShared);
+  }
+
+  @Test
+  public void findCommonRemovesDisagreeingKeys() {
```

Comment:
A better name might be "findCommonRemovesKeysWithDisagreeingValues"

Reply:
Sure would be!